src/components/Atoms/EditButton/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styled from "styled-components";
+import pencil from "../../../assets/svg/pencil.svg";
+import COLOR from "../../../variables/color";
+
+const EditButton = (props) => {
+  return (
+    <Button onClick={props.onClick}>
+      <Circle></Circle>
```

Comment:
```suggestion
      <Circle/>
```
中身のない空要素の場合はこのようにもかけます。